### ISSUE TITLE
.

### ISSUE THREAD

**Issue by shaun-ba** [sqlitebrowser/sqlitebrowser#1393]
.

**Comment by justinclift** [comment]
Oh, that sounds weird.

Does it happen consistently, even across restarts of DB4S?

As a thought, would you be ok to try the nightly builds for Ubuntu and see if that's better? :smile:

&nbsp; &nbsp; https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds-1

**Comment by shaun-ba** [comment]
Yes very odd and does happen on restarts too. I can try the nightly builds
tomorrow but will that fix if a fix hasn't specifically been made for the
issue? I do hate trying new code "just because"

**Comment by justinclift** [comment]
Oh, I was definitely recommending it for "just because".  We're getting kind of near a next release, and have lots of bug fixes in the nighty builds compared to the last release.  But nothing springs to mind as definitely this issue.

On the other hand, I'm kind of wondering if the fix here #1168 (only in our nightly builds so far) might have any effect. :smile:

**Comment by lucydodo** [comment]
@shaun-ba Is there a reason why you changed the issue title? Could it have been a mistake?

**Comment by justinclift** [comment]
Meh, I'm closing this.  It looks like @shaun-ba deleted the issue report on purpose so there's no point in keeping it open. :wink:
